.github/workflows/continuous_testing.yml

```diff
@@ -0,0 +1,40 @@
+on:
+  schedule:
+    - cron: '30 18 * * *'
+
+jobs:
+  test_live:
+    name: Station Tests Live
+    runs-on: ubuntu-latest
+    if: ${{ github.ref == 'refs/heads/main' }}
+    env:
+      DISCORD_ACCESS_TOKEN: ${{ secrets.DISCORD_ACCESS_TOKEN }}
+      SLACK_ACCESS_TOKEN: ${{ secrets.SLACK_ACCESS_TOKEN }}
+    steps:
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          registry-url: https://registry.npmjs.org/
+          scope: '@superfaceai'
+          node-version: '14'
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v2.1.3
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - run: yarn install
+      - run: yarn build_reporter
+      - run: sudo yarn hosts:allow
+      - run: cat /etc/hosts
+      - run: yarn run check
+      - run: yarn lint
+      - run: yarn structure
```

Comment:
These steps are not needed here. Nothing with wrong structure, not linted, will get to main.